src/project/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .user import User
+from .event import Event
```

Comment:
Why didn't you complete it like in KnowledgeHub?

`
__all__ = [
    "User",
    "Event"
]
`

like this